test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -481,3 +481,37 @@ test('we should clean up hammerhead script and style for all elements(GH-1079)',
     ok(checkProperty(innerHTML));
     ok(checkProperty(outerHTML));
 });
+
+if (browserUtils.isIE && !browserUtils.isMSEdge) {
+    test('we should clean up html and remove extra namespaces from svg (GH-1083)', function () {
```

Comment:
Change checking algorithm:
1)create native element, set `xmlns:link` attribute for this, calculate href for `a` tag
2)create overriden element, set ....
3)check that native element markup equals the processed markup